### ISSUE TITLE
Error executing migration 2022_03_21_000001_users_autologin

### ISSUE THREAD

**Issue by elkangaroo** [munkireport/munkireport-php#1470]
We updated munkireport based on the Upgrade Procedure in the [Wiki](https://github.com/munkireport/munkireport-php/wiki/General-Upgrade-Procedures), but the Database migrations throw an error:

**$ ./please migrate**

```
Migrating: 2022_03_21_000001_users_autologin

In Connection.php line 671:
  SQLSTATE[42S02]: Base table or view not found: 1146 Table 'munkireport.users' doesn't exist (SQL: alter table `users` add `autologi  
  n_enabled` tinyint(1) null)                                                                                                          

In Exception.php line 18:
  SQLSTATE[42S02]: Base table or view not found: 1146 Table 'munkireport.users' doesn't exist  

In PDOStatement.php line 117:
  SQLSTATE[42S02]: Base table or view not found: 1146 Table 'munkireport.users' doesn't exist  
```

This seems to be the related PR from "users" module: https://github.com/munkireport/users/pull/12 (was just released yesterday)

Look like the table is named "local_users" and not "users". I ran the SQL statements with corrected table name manually and it seems to be working.

**Comment by tuxudo** [comment]
This has been resolved with version v2.3 of the users module.

**Comment by elkangaroo** [comment]
Thanks for the fast reply + fix, it is solved :)
